test/JDBC/expected/SELECT_INTO_TEST-vu-verify.out

```diff
@@ -0,0 +1,300 @@
+
+-- create a new table select_into 1
+create table select_into(select_into_COL int);
+go
+select select_into_COL from select_into;
+go
+~~START~~
+int
+~~END~~
+
+-- Create a new table select_into_REPO with column select_into_COL 
+select select_into_COL into select_into_REPO from select_into;
+go
+-- Indexing over column select_into_COL
+create index IDX_REPRODUCTION on select_into_REPO(select_into_COL);
```

Comment:
Let's also add test cases to test 
1. ALTER TABLE...ALTER COLUMN works fine on this column
2. Add new constraint through ALTER TABLE ... ADD CONSTRAINT
3. CREATE UNIQUE INDEX
4. ALTER TABLE ... DROP COLUMN